core/src/main/kotlin/fr/sncf/osrd/sim/ASyncDES.kt

```diff
@@ -0,0 +1,88 @@
+package fr.sncf.osrd.sim
```

Comment:
`ASyncDES.kt` is an obscure file name, maybe we could rename it to `DiscreteEventSimulation`? I don't think there will be one that isn't async.